docs/dapp/emerald/README.mdx

```diff
@@ -1,107 +1,96 @@
+- Chain ID:
+  - Hex: `0xa516`
+  - Decimal: `42262`
 
-The reward program is two years long.
+### Testnet
 
-## Web3 Gateway
+- Network name: `emerald-testnet`
+- Long network name: `Oasis Emerald Testnet`
+- Chain ID:
+  - Hex: `0xa515`
+  - Decimal: `42261`
 
-To get started building on our Emerald ParaTime, you can use our public Web3 gateway, fully compatible with Ethereum's Web3 gateway.
+## RPC Endpoints
 
-### Mainnet
+:::danger
 
-* RPC HTTP endpoint: `https://emerald.oasis.dev`
-* RPC WebSockets endpoint: `wss://emerald.oasis.dev/ws`
-* Chain ID:
-  * Hex: 0xa516
-  * Decimal: 42262
-* Block explorer: [https://explorer.emerald.oasis.dev](https://explorer.emerald.oasis.dev)
-
-<button
-  class="button button--primary margin-bottom--md"
-  onClick={() => {
-    if (!window.ethereum?.request) {
-      return alert('Have you installed MetaMask yet? If not, please do so.\n\nComputer: Once it is installed, you will be able to add the ParaTime to your MetaMask.\n\nPhone: Open the website through your MetaMask Browser to add the ParaTime.')
-    }
-    window.ethereum.request({
-      method: 'wallet_addEthereumChain',
-      params: [
-        {
-          chainId: '0xa516',
-          chainName: 'Oasis Emerald',
-          nativeCurrency: {
-            name: 'Emerald Rose',
-            symbol: 'ROSE',
-            decimals: 18,
-          },
-          rpcUrls: ['https://emerald.oasis.dev', 'wss://emerald.oasis.dev/ws'],
-          blockExplorerUrls: ['https://explorer.emerald.oasis.dev'],
-        },
-      ],
-    })
-  }}
->
-  Click here to register Emerald Mainnet to your MetaMask or Brave Wallet
-</button>
+The RPC endpoint is a *point of trust*. Beside traffic rate limiting, it can
+also perform censorship or even a man-in-the-middle attack. If you have security
+considerations, we strongly recommend that you set up your own [ParaTime client
+node][paratime-client-node] and the [Web3-compatible gateway].
 
-### Testnet
+:::
+
+
+[Web3-compatible gateway]: ../../node/web3.md
+[paratime-client-node]: ../../node/run-your-node/paratime-client-node.mdx
+
+You can connect to one of the public Web3 gateways below (in alphabetic order):
+
+| Provider                    | Mainnet RPC URLs                                                        | Testnet RPC URLs                                                                         |
+|-----------------------------|-------------------------------------------------------------------------|------------------------------------------------------------------------------------------|
+| [1RPC]                      | <E rpcs={['https://1rpc.io/oasis/emerald']} />                          | N/A                                                                                      |
+| [Oasis Protocol Foundation] | <E rpcs={['https://emerald.oasis.dev','wss://emerald.oasis.dev/ws']} /> | <ET rpcs={['https://testnet.emerald.oasis.dev','wss://testnet.emerald.oasis.dev/ws']} /> |
```

Comment:
Are these still only on .dev? @ptrus

Reply:
I didn't find any tasks/issues mentioning moving the emerald endpoints to io as well (only for grpc and sapphire). I can look into moving those as well.

Reply:
Mainnet one is blocked on an external operator, did reach out few days ago (for a different reason) no response yet.

Reply:
Any news on migrating testnet.emerald.oasis.dev and emerald.oasis.dev domains?

Reply:
`emerald.oasis.io` and `testnet.emerald.oasis.io` now work